src/tui.rs

```diff
@@ -393,7 +398,8 @@ impl ToipeTui {
 
                 // clear line
                 line = Vec::new();
-                current_len = 0;
+                line.push(word.clone());
+                current_len = word.len() as u16 + 1;
```

Comment:
One question about this - does this mean that a word was being skipped after a line was completed?

Reply:
yeah, when the terminal was too small, the number of words were like less than 10. so i looked through the code and found that each time when a line is completed, the current word was skipped(which caused the <10 words issue). so yeah i also fixed this minor bug

Reply:
nice! thanks